ras_party/views/account_view.py

```diff
@@ -0,0 +1,82 @@
+from flask import Blueprint, request, current_app, make_response, jsonify
+from flask_httpauth import HTTPBasicAuth
+
+import ras_party.controllers.account_controller
+import ras_party.controllers.business_controller
+import ras_party.controllers.party_controller
+import ras_party.controllers.respondent_controller
+from ras_party.support.log_decorator import log_route
+
+account_view = Blueprint('account_view', __name__)
+
+
+auth = HTTPBasicAuth()
+
+
+@auth.get_password
+def get_pw(username):
+    config_username = current_app.config['SECURITY_USER_NAME']
+    config_password = current_app.config['SECURITY_USER_PASSWORD']
+    if username == config_username:
+        return config_password
+
+
+@account_view.route('/respondents/change_email', methods=['PUT'])
+@account_view.route('/respondents/email', methods=['PUT'])
+@auth.login_required
```

Comment:
If we expect to `@auth.login_required` for every api in this blueprint we might be able to use http://flask.pocoo.org/docs/0.12/api/#flask.Flask.before_request. Same with `@log_route`